### PR TITLE
Documentation/access controls ac 3

### PR DESCRIPTION
**Pull request by rnafarrete**
### This pull request changes...

first draft of implementation statement for ac-3 access enforcement

### This pull request is ready to merge when...

- [ ] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [ ] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

File: docs/controls/access_control/ac-3.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Access Control
+## AC-03 - ACCESS ENFORCEMENT
+
+The information system enforces approved authorizations for logical access to information and system resources in accordance with applicable access control policies.  
+
+### TDP Implementation
+Access enforcement to the TDP application is provided by Login.gov.  All users will login through Login.gov and MFA to gain access to the TDP application.
```
Comment (carltonsmith) [inline docs/controls/access_control/ac-3.md:7]: Instead of "MFA" I would say "Multi-factor Authentication (MFA)"

**Review by carltonsmith — CHANGES_REQUESTED** [review]
Thanks again! Just one small change

**Comment by rnafarrete** [comment]
Updated implementation statement per feedback.

**Review by carltonsmith — APPROVED** [review]
LGTM!

**Comment by carltonsmith** [comment]
PR submitted to HHS/main

**Comment by hursey013** [comment]
@carltonsmith As we discovered in https://github.com/HHS/TANF-app/pull/176 it looks like AC-3 is not required as part of the customer responsibility matrix. Let's keep the branch around though in the event it turns out it is needed in the future.

**Pull request closed without merging**